docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -89,44 +83,58 @@ Next, you need to remove the empty users from your MySQL database:
 
 Next, you will need to configure each server so the services can bind on IP addresses they don't currently have configured. This allows faster failover of the services.
 
-On all your servers, add the following line in `/etc/sysctl.conf` and then reload with 'sysctl -p'
+On *all your servers*, add the following line in `/etc/sysctl.conf` and then reload with 'sysctl -p'
 
   net.ipv4.ip_nonlocal_bind = 1
 
 Then, disable the default MariaDB systemd service unit as PacketFence already provides one (packetfence-mariadb)
 
   # systemctl disable mariadb
 
-Now, on the first server, create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates. What matters is that the path to the PEM file should be /usr/local/pf/conf/ssl/server.pem since HAProxy will expect it to be there.
+Now, on *the first server*, create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates. What matters is that the path to the PEM file should be /usr/local/pf/conf/ssl/server.pem since HAProxy will expect it to be there.
 
   # cd /usr/local/pf/conf/ssl
   # cat server.key server.crt > server.pem
 
-Going through the configurator
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Basic PacketFence configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Now, on the first server of your cluster, you should go through the configurator, configuring the server as if it was in standalone (no HA interfaces). You should leave the services stopped at the end of the configurator.
+Now, on *the first server* of your cluster, you should go through the configurator, until *step 6* (Fingerbank configuration). You should leave the services stopped at the end of the configurator.
 
-Then restart PacketFence's mariadb on the first server:
+Then restart PacketFence's mariadb on *the first server*:
 
   # systemctl restart packetfence-mariadb
 
-On the other servers of your cluster, configure only the network interfaces without going past that section in the configurator. If the other servers already have the right IP addresses configured on their interfaces, you can ignore this step.
+On the *other servers* of your cluster, configure only the network interfaces (step 2) without going past that section in the configurator. If the other servers already have the right IP addresses configured on their interfaces, you can ignore this step.
 
-Step 3 : Create a new cluster
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+At this point, for a VLAN enforcement configuration for example, the network interfaces of your servers must be configured, and you must be able to see, for each server:
+
+|===
+|In `/ect/sysconfig/network-scripts/`| 
+|One Management Interface|ifcfg-*YourFirstInterfeceName*
```

Comment:
I don't really want to know your feces names

interfece->interface

Reply:
inter-💩